src/woo_search/search_index/tasks/publications.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from woo_search.celery import app
+
+from ..client import get_client
+from ..documents import Document
+from ..typing import PublisherType
+
+
+@app.task()
+def index_document(
+    uuid: str,
+    publicatie: str,
+    publisher: PublisherType,
+    identifier: str,
+    officiele_titel: str,
+    verkorte_titel: str,
+    omschrijving: str,
+    creatiedatum: datetime,
+    registratiedatum: datetime,
+    laatst_gewijzigd_datum: datetime,
+):
+    document = Document(
+        _id=uuid,
+        uuid=uuid,
+        publicatie=publicatie,
+        publisher=publisher,
+        identifier=identifier,
+        officiele_titel=officiele_titel,
+        verkorte_titel=verkorte_titel,
+        omschrijving=omschrijving,
+        creatiedatum=creatiedatum,
```

Comment:
```suggestion
        creatiedatum=datetime.combine(creatiedatum, time.min, tzinfo=<ams timezone>),
```
See https://github.com/open-formulieren/open-forms/blob/a1a1b0048ec8bda03bb50631ec2bc176fee9be2f/src/openforms/utils/date.py#L14 and https://github.com/open-formulieren/open-forms/blob/a1a1b0048ec8bda03bb50631ec2bc176fee9be2f/src/openforms/utils/date.py#L76 for some inspiration